src/components/Movies.jsx

```diff
@@ -0,0 +1,22 @@
+import Movie from './Movie'
+import '../styles/movies.scss'
+
+const Movies = ({ movies, viewTrailer, closeCard }) => {
+
+    return (
+        <div data-testid="movies">
```

Comment:
Creating a MoviesGrid component could be useful for reusing it to display both the "Starred" list and the "Watch Later" list.